build.sbt

```diff
@@ -3,22 +3,22 @@ import sbtrelease.ReleasePlugin.autoImport.ReleaseTransformations._
 
 
 // Basics
-val typesafe = "Typesafe Repository" at "https://repo.typesafe.com/typesafe/releases/"
-val typesafeSnapshot = "Typesafe Snapshots Repository" at "https://repo.typesafe.com/typesafe/snapshots/"
+val apacheSnapshot = "Apache Snapshots".at("https://repository.apache.org/content/repositories/snapshots/")
 val sonatypeSnapshot = "Sonatype Snapshots Repository" at "https://oss.sonatype.org/content/repositories/snapshots/"
+//todo resolvers += Resolver.jcenterRepo
+//todo resolvers += Resolver.ApacheMavenSnapshotsRepo
 
 val mainScalaVersion = "3.1.3" // note: Scala 3.2 is not forward compatible - publishing with 3.2 would force users to Scala 3.2
```

Comment:
Since Pekko is using Scala 3.3 we can also upgrade as users anyway have to use it.